build.gradle

```diff
@@ -0,0 +1,56 @@
+plugins {
+  id 'maven-publish'
+}
+
+publishing {
+
+  publications {
+    maven(MavenPublication) {
+      groupId = 'com.github.pseudomuto'
```

Comment:
I used io.github.pseudomuto...should've looked closer before creating that ticket 😬

Reply:
No problem, will quickly update 😄